### ISSUE TITLE
Posibilitly or option to remove app from recent apps.

### ISSUE THREAD

**Issue by kuriel-trivu** [wumke/react-native-exit-app#52]
When using RNExitApp.exitApp(); there is no way to remove app from recent apps.

**Comment by wumke** [comment]
Indeed, this is not implemented. In case anyone finds a way you may implement it by using an additional parameter in the current function and create a pull request. I will merge it.

**Comment by kuriel-trivu** [comment]
This is definitively possible. I removed this package because today is not our desired action and seems that @wumke won't take any action.

**Comment by kartavyaparekh96** [comment]
how it is possible @kuriel-trivu ?

**Comment by wumke** [comment]
@kuriel-trivu As I mentionned: in case you want changes, you are free to create a pull request (functionality should be implemented for both platforms: Android and iOS, else I will not merge it).

As @kartavyaparekh96 already asked: explain how to do it if you know it... I do not have spare time to spend time investigating needs for other people's projects, but I'm always interest in usefull improvements.

**Comment by Nantris** [comment]
@kuriel-trivu would you mind sharing your findings? This would be a great help to the community.
